### PR TITLE
[SDK] Detect appURI

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: packages/sdk/src/evm/contracts/smart-contract.ts [hunk 1]
```diff
@@ -286,4 +292,11 @@ export class SmartContract<TContract extends BaseContract = BaseContract>
     }
     return undefined;
   }
+
+  private detectAppURI() {
+    if (detectContractFeature<AppURI>(this.contractWrapper, "AppURI")) {
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/contracts/smart-contract.ts:297]: Here you should also check for the contract metadata extension

Comment (nachoiacovino) [reply packages/sdk/src/evm/contracts/smart-contract.ts:297]: so check for both? you can have appURI without contract metadata

Comment (joaquim-verges) [reply packages/sdk/src/evm/contracts/smart-contract.ts:297]: one OR the other.

`appUri = detect(AppUri) || detect(ContractMetadata)`

Comment (joaquim-verges) [reply packages/sdk/src/evm/contracts/smart-contract.ts:297]: @nachoiacovino @furqanrydhan ^

**Pull request merged**